content/first-order-logic/models-theories/theories.tex

```diff
@@ -126,8 +126,8 @@
         \Part{y}{z}) \lif \Part{x}{z})]]],\\
 \intertext{Moreover, any two objects have a fusion (an object that has
   only these two objects and all their parts as parts).}  &
-\lforall[x][\lforall[y][\lexists[z][\lforall[u][(\Part{u}{z} \liff
-        (\Part{u}{x} \land \Part{u}{y}))]]]]
+\lforall[x][\lforall[y][\lexists[z][\lforall[u][(\Part{z}{u} \liff
```

Comment:
No. Note that is parthood is like 'subset of' and not 'element of'. If we put in an 'or' then we had Pzx or Pzy since Pzz is true and on the other hand Pxz and Pyz; thus either z=x or z=y; certainly not what we want. I think the fusion is like a join in a poset, i.e. the thing that is exactly part of all the things having x and y as parts. Currently, before changing the order, it would be the product, like the meet in a poset. I am no expert on mereology, I just did a quick search. Actually the fusion should only exist if x and y underlap, i.e. there is an upper bound for x and y. As this is just an example in the book, I was not sure if this should be included or not.